feature/bgp/policybase/otg_tests/extcomm_action_test/README.md

```diff
@@ -0,0 +1,176 @@
+    * statement[name='add_std_comms']/
+      * actions/bgp-actions/set-community/reference/config/community-set-refs =
+          /routing-policy/defined-sets/bgp-defined-sets/community-sets/community-set[name='add_std_comms']
+      * actions/bgp-actions/set-community/config/options = ADD
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition[name='match_and_add_comms'/`
+    with the following `statements/`
+    * statement[name='match_and_add_std_comms']/
+      * conditions/bgp-conditions/match-community-set/config/community-set =
+        /routing-policy/defined-sets/bgp-defined-sets/community-sets/community-set[name='match_std_comms']
+      * actions/bgp-actions/set-community/reference/config/community-set-refs =
+          /routing-policy/defined-sets/bgp-defined-sets/community-sets/community-set[name='add_std_comms']
+      * actions/bgp-actions/set-community/config/options = ADD
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * For each policy-definition created, run a subtest (RT-7.8.2.x-neighbor-<policy_name_here>) to
+    * Use gnmi Set REPLACE option for:
+      * `/routing-policy/policy-definitions` to configure the policy
+      * Use `/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/apply-policy/config/import-policy`
+        to apply the policy on the DUT bgp neighbor to the ATE port 1.
+    * Verify routes are received on ATE port 1 for accepted prefixes.
+    * Verify expected communities are present.
+
+  * For each policy-definition created, run a subtest (RT-7.8.2.x-peer-group-<policy_name_here>) to
+    * Use gnmi Set REPLACE option for:
+      * `/routing-policy/policy-definitions` to configure the policy
+      * Use `/network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/apply-policy/config/import-policy`
+        to apply the policy on the DUT bgp neighbor to the ATE port 1.
+    * Verify routes are received on ATE port 1 for accepted prefixes.
+    * Verify expected communities are present.
+
+    * Expected communities
+      |              | add_std_comms                                 | match_and_add_std_comms           |
+      | ------------ | --------------------------------------------- | --------------------------------- |
+      | prefix-set-1 | [ 10:10, 20:20, 30:30 ]                       | none                              |
+      | prefix-set-2 | [ 10:10, 20:20, 30:30, 5:5, 6:6 ]             | [ 10:10, 20:20, 30:30, 5:5, 6:6 ] |
+      | prefix-set-3 | [ 10:10, 20:20, 30:30, 50:500000, 60:600000 ] | [ 50:500000, 60:600000 ]          |
+
+* RT-7.8.3 - Create policy to set ext-community for various match criteria using OC release 2.x
```

Comment:
This entire test is not needed.
- The communities used ARE NOT legal communities of any know type.
  - They are not standard communities, because Local admin part is >65535
  - They are not large comunities (rfc8092) because violates chapter 5 of this rfc
  - They are not extended communities, because extended community MUST have designated type
- The only ext-communities we care is link-bandwidth, which is tested in separate test.